src/Components/ExpandedRulesDetails.js/ExpandedRulesDetails.js

```diff
@@ -20,11 +20,20 @@ import PropTypes from 'prop-types';
 import TemplateProcessor from '@redhat-cloud-services/frontend-components-advisor-components/TemplateProcessor/TemplateProcessor';
 import ObjectsModal from '../ObjectsModal/ObjectsModal';
 import { ObjectsTableColumns } from '../../AppConstants';
+import isEmpty from 'lodash/isEmpty';
 
 const code = `oc get namespace -o jsonpath={range .items[*]}{.metadata.name}{"\t"}{.metadata.uid}{"\n"}{end}
   oc -n <namespace> get <resourceKind> -o jsonpath={range .items[*]}{.metadata.name}{"\t"}{.metadata.uid}{"\n"}{end}`;
 
-const ExpandedRulesDetails = ({ more_info, resolution, objects }) => {
+const ExpandedRulesDetails = ({
+  more_info,
+  resolution,
+  objects,
+  namespaceName,
+  reason,
+  extra_data,
+}) => {
+  const objectsArePresent = !isEmpty(objects);
```

Comment:
Just a small note, maybe we could make the bundle a bit smaller by removing the lodash function. I believe it always an array so we could check it like this
```suggestion
  const objectsArePresent = Array.isArray(objects) && objects.length > 0;
```